7_react/react1/week2/TODO-LIST/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import toDoItems from "./component/TodoItems";
+import Timer from "./component/timer";
+import "./App.css";
+
+
+function App() {
+  const [toDoItem, setToDoItems] = useState(toDoItems);
+// Function to add a random item
+  const addRandomItem = () => {
+    const randomId = Math.floor(Math.random() * 1000);
+    const newItem = {
+      id: randomId,
+      description: `Task ${randomId}`,
+      deadline: "Some deadline", 
+      done: false // Initially task is not done
+    };
+    setToDoItems([...toDoItems, newItem]);
```

Comment:
Since `toDoItems` is the const array that you imported, this method would always just set the new state to be the original array plus the **latest** new to-do. However, as noted in the previous comment, if you just make sure that `toDoItems` is the actual state variable, this code will be perfectly fine! 🧠